src/applications/vaos/tests/new-appointment/components/ContactInfoPage.unit.spec.jsx

```diff
@@ -1,106 +1,116 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import { expect } from 'chai';
-import sinon from 'sinon';
-import { mount } from 'enzyme';
+import userEvent from '@testing-library/user-event';
 
-import { fillData } from 'platform/testing/unit/schemaform-utils.jsx';
-import { ContactInfoPage } from '../../../new-appointment/components/ContactInfoPage';
+import ContactInfoPage from '../../../new-appointment/components/ContactInfoPage';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { FETCH_STATUS } from '../../../utils/constants';
 
 describe('VAOS <ContactInfoPage>', () => {
-  it('should render', () => {
-    const openFormPage = sinon.spy();
-    const updateFormData = sinon.spy();
-
-    const form = mount(
-      <ContactInfoPage
-        openFormPage={openFormPage}
-        updateFormData={updateFormData}
-        data={{}}
-      />,
-    );
-
-    expect(form.find('input').length).to.equal(5);
-    form.unmount();
+  it('should render', async () => {
+    const store = createTestStore({
+      newAppointment: {
+        pages: [],
+        previousPages: [],
+      },
+    });
+
+    const screen = renderWithStoreAndRouter(<ContactInfoPage />, {
+      store,
+    });
+
+    // it should display page heading
+    expect(await screen.findByText('Your contact information')).to.be.ok;
```

Comment:
This test isn't really doing much. I'd say maybe move the ` expect(screen.findByText(/^Please enter a phone number/)).to.be.ok;` calls up here from the below test, or incorporate this expect statement in the next test.

Also, if it's helpful, I tried to outline the tests we normally write for each form page in [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/engineering/testing/fe_testing_guidance.md#what-types-of-things-should-i-test):

- The page renders with the correct number of form elements
- The page displays validation messages for required data if you submit without entering them
- The page submits successfully if all required info is filled out
- If you unmount and remount the page with the same store, the previously entered form data is still there

Reply:
Done